Woof/Woof/View/SitterFlow/SitterProfileView/SitterProfileView.swift

```diff
@@ -19,22 +20,27 @@ struct SitterProfileView: View {
                     )
 
                     HStack {
-                        Button("Save") {
-                            viewModel.save()
-                            isEditingMode.toggle()
-                        }
+                        if isEditingMode {
+                            Button(cancelButtonLabelText) {
+                                cancelEditing()
```

Comment:
```suggestion
                                viewModel.cancelEditing()
```
it should look something like this.

Reply:
Fixed.